code-for-canada/project-thundercat
# [Add Task] Added Tooltips in 'Add Task' form

# Description

The tooltip icons were already in the _Add Task_ form, but they were static, so I added the tooltips content using _react-tooltip_ library, since the bootstrap one wasn't working well.

## Type of change

- New feature (non-breaking change which adds functionality)
- Dependency change

## Screenshot

**Task Tooltip (EN):**

![image](https://user-images.githubusercontent.com/23021242/56155724-95ce6300-5f89-11e9-81c5-a50c5bc3822d.png)

**Task Tooltip (FR):**

![image](https://user-images.githubusercontent.com/23021242/56155754-a383e880-5f89-11e9-8653-64bf6ab556cd.png)

**Reasons For Action Tooltip (EN):**

![image](https://user-images.githubusercontent.com/23021242/56155812-c6ae9800-5f89-11e9-87c5-b8c3d0057c45.png)

**Reasons For Action Tooltip (FR):**

![image](https://user-images.githubusercontent.com/23021242/56155793-ba2a3f80-5f89-11e9-8861-f9589b5879db.png)

# Testing

Manual steps to reproduce this functionality:

1.  Start the eMIB Sample Test.
2.  Complete the _pre-test_ process.
3.  Open the _Inbox_ tab.
4.  Click on _Create a task_.
5.  Make sure you are able to see the tooltips when navigating using the keyboard and when clicking on the icons.

# Checklist

Applicable for all code changes.
